dockerize.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+IMAGE_NAME="boresha/agriculture-core"
+DOCKER_HOME="/agiculture-core/"
+
+if [ "$1" == "init" ]; then
+	docker build -t $IMAGE_NAME .
```

Comment:
What was wrong with `$DIR`? It let's us run this script from other directories.

Reply:
I changed it to be consistent with the dashboard dockerize.sh script. the $DIR variable was set pointing to the pwd anyway

Reply:
They should both use `$DIR`, the thing I have at the top is pretty robust way of getting the dir of the script.

Reply:
Okay it has been changed to $DIR